pkg/store/instance.go

```diff
@@ -20,7 +20,7 @@ import (
 
 	"github.com/docker/go-units"
 	hostagentclient "github.com/lima-vm/lima/pkg/hostagent/api/client"
-	"github.com/lima-vm/lima/pkg/identifierutil"
+	instancehostname "github.com/lima-vm/lima/pkg/instance/hostname"
```

Comment:
I guess here we import the other instance package? this shows that we should not call the new package "instance".